src/components/esn-app-grid/esn-app-grid.tsx

```diff
@@ -0,0 +1,103 @@
+import { Component, Prop, ComponentInterface, State, Host, h } from '@stencil/core';
+import { Application } from './esn-app-grid.types';
+import { AppGridTogglerIcon } from './icons/AppGridIcon';
+import { CalendarAppIcon } from './icons/CalendarAppIcon';
+import { InboxAppIcon } from './icons/InboxAppIcon';
+import { AdminAppIcon } from './icons/AdminAppIcon';
+import { ContactsAppIcon } from './icons/ContactsAppIcon';
+import { LinShareAppIcon } from './icons/LinShareAppIcon';
+import { UnknownAppIcon } from './icons/UnknownAppIcon';
+
+const iconMapping = {
+  Admin: AdminAppIcon,
+  Calendar: CalendarAppIcon,
+  Contacts: ContactsAppIcon,
+  Inbox: InboxAppIcon,
+  LinShare: LinShareAppIcon
+};
```

Comment:
Even if temporary, this is bad to have this mapping defined in a component

Reply:
Fixed. Now there's an util to get the app icon.